cal/cal/static/js/app.js

```diff
@@ -3,32 +3,120 @@ var analyticsApp = angular.module('analyticsApp', ['nvd3', 'ui.calendar']);
 analyticsApp.controller('LoggedInCtrl', function LoggedInController($scope) {
 });
 
-// Controller to generate graph data from chart directive, cumulative tags
 analyticsApp.controller('TagsCtrl', function($scope, $http){
   var url = '/v1/tags.json';
+  var tagUrl = '/v1/tags/';
+
   $scope.url = url;
```

Comment:
Not sure why I added this before, but could you delete it? From a cursory glance it doesn't look like this is being used